packages/gateway-interface/src/index.ts

```diff
@@ -118,6 +118,10 @@ export interface GatewayGraphQLRequestMetrics {
   registeredOperation?: boolean;
   startHrTime?: [number, number];
   queryPlanTrace?: Trace.QueryPlanNode;
+  nonFtv1ErrorPaths?: {
```

Comment:
unrelated: the comment at the very top of this file is outdated

Reply:
[12355d9](https://github.com/apollographql/apollo-server/pull/7136/commits/12355d927660a24ad744e784158948e63d9c8931)